var/spack/repos/builtin/packages/neurodamus-asciitoh/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+from spack import *
+import shutil
+import os
+
+class NeurodamusAsciitoh(Package):
```

Comment:
How about `ascii2h`?

Reply:
Also note that the URL has an additional 5 in there. Is this ASCII to headers?

Reply:
`ascii2h` sounds nicer.
I think I had issues when a spack package ended with a number. It was meant to be `asciitoh5` but I would get strange errors I don't exactly recall when I created it. Can you confirm if there's any issues with creating a package whose name ends with a number?

Reply:
Also, it's meant to be `asciitohdf5`.

Reply:
I don't see why there should be issues. Just ran across a package called `py-rpy2` [sic].

`asciitoh5` would be in line with the repository URL.

Reply:
> asciitoh5 would be in line with the repository URL

+2

> I think I had issues when a spack package ended with a number. It was meant to be asciitoh5 but I would get strange errors I don't exactly recall when I created it.

I don't think this is related. If you an error, could you post?

Reply:
I will recreate it as `asciitoh5` then.

Reply:
This is the issue I'm having: 
```
 spack create neurodamus-asciitoh5     
==> This looks like a URL for neurodamus-asciitoh
==> Warning: Spack will not check SSL certificates. You need to update your Python to enable certificate verification.
==> Warning: Spack will not check SSL certificates. You need to update your Python to enable certificate verification.
==> Found 1 version of neurodamus-asciitoh:
  
  5  neurodamus-asciitoh5

==> How many would you like to checksum? (default is 1, q to abort) 1
==> Downloading...
==> Fetching neurodamus-asciitoh5

curl: (6) Could not resolve host: neurodamus-asciitoh5; Name or service not known
==> Fetching from neurodamus-asciitoh5 failed.
==> Something failed on neurodamus-asciitoh5, skipping.
    (All fetchers failed for spack-stage-SlmBor)
==> Error: Could not fetch any versions for neurodamus-asciitoh
```

Reply:
```
curl: (6) Could not resolve host: neurodamus-asciitoh5
```
as error says, `curl` command  is trying to download from web server name `neurodamus-asciitoh5`. (which is wrong). Lets check offline.